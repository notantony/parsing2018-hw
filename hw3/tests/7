abs a b =
if >= 0 - a b
    print - a b
    print - b a