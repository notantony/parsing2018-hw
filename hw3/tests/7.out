void abs(int a, int b) {
    if ((0 >= (a - b))) {
        std::cout << (a - b);
    }
    else {
        std::cout << (b - a);
    }
}
